lib/common.sh

```diff
@@ -112,11 +112,16 @@ export DEFAULT_HOSTS_MEMORY=${DEFAULT_HOSTS_MEMORY:-8192}
 # Cluster.
 export CLUSTER_NAME=${CLUSTER_NAME:-"test1"}
 export CLUSTER_APIENDPOINT_IP=${CLUSTER_APIENDPOINT_IP:-"192.168.111.249"}
-export KUBERNETES_VERSION=${KUBERNETES_VERSION:-"v1.17.0"}
-export KUBERNETES_BINARIES_VERSION="${KUBERNETES_BINARIES_VERSION:-${KUBERNETES_VERSION}}"
+
+export KUBERNETES_VERSION=${KUBERNETES_VERSION:-"v1.18.0"}
```

Comment:
this is broken wrt. the latest PR that went in. the version should not be modified (Anwar has a fix to upgrade to 1.18.0, it will not work before that) and you need to keep the KUBERNETES_BINARIES_VERSION variable

Reply:
Fixed